uber/site_sections/summary.py

```diff
@@ -157,14 +157,21 @@ def printed_badges_staff(self, out, session):
 
         # part 1, include only staff badges that have an assigned name
         uber.reports.PersonalizedBadgeReport().run(out, session,
-            sa.Attendee.badge_type == c.STAFF_BADGE,
-            sa.Attendee.badge_num != None,
+            Attendee.badge_type == c.STAFF_BADGE,
+            Attendee.badge_num != None,
             order_by='badge_num')
 
-        # part 2, include a bunch of extra badges so we have some printed
+        # part 2, include unassigned group badges that are staff.  these don't have badge numbers
+        minimum_safety_amount = 5
```

Comment:
Sorry, not quite right here!

What we'd want is:
Always a minimum of 10 'extra' badges
The unassigned badges + 5

So what you'd wanna do here is take out this var, then after your extra_count down below, add:

`extra_count = min(10, extra_count + 5)`

And then you can take out the `max()` line in the final `badge_range` assignment